banzai_nres/pdf.py

```diff
@@ -86,24 +87,25 @@ def do_stage(self, image):
         line_separation, top_line = 0.065, 0.925
         pl.text(0.1, top_line, image.meta['ORIGNAME'].replace('e00', 'e' + str(self.runtime_context.reduction_level) +
                                                               '-1d') + '.fz')
+        pl.text(0.1, top_line - line_separation, f'Request Numbers: {image.request_number}')
```

Comment:
Not shown yet in the attached example pdf. But nice! 

Would it make sense to hyperlink this, or is the url too volatile to bake in here?